charts/fybrik/files/adminconfig/infrastructure.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Shouldn't there be an empty Infrastructure in the core Fybrik, and one with values in samples?  Also, perhaps the infrastructure should sit outside of adminconfig since it will be used by other components as well - ex: to allocate storage from within the manager.